Disabling root user and adding default user

rahulait opened the pull request.

rahulait committed:
Merge pull request #3 from CCI-MOC/master

Merge pull request #7 from rahulait/master

rahulait committed:
Disabling root access and adding default user.

rahulait committed:
Missed files for previous commit.

lokI8 commented:
Looks good to me. As long as we add the two params to the heira file, this looks ready to merge. Good job =)

+1

rahulait committed:
Added support to store user-passwd in home-directory.

rahulait committed:
Database name changes.

For neutron/compute.pp, since its directly modifying the
neutron.conf file, we had to add extra variables which will
be set in the hiera.yaml file. For others, we use the values
which we set in db/mysql.pp file.

ravisantoshgudimetla commented:
+1, Looks good to me.

lokI8 commented:
Merging
